pkgs/tools/misc/adr-tools/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchzip }:
+
+stdenv.mkDerivation rec {
+  name = "adr-tools-${version}";
+  version = "3.0.0";
+
+  src = fetchzip {
+    url = "https://github.com/npryce/adr-tools/archive/${version}.tar.gz";
+    sha256 = "1igssl6853wagi5050157bbmr9j12703fqfm8cd7gscqwjghnk14";
+  };
+
+  buildInputs = [];
+
+  dontBuild = true;
```

Comment:
As we privately discussed, the package runs tests when running `make`.

If you can get it working I think it would be good to run them in the test phase.